src/services/iqPrice/index.ts

```diff
@@ -15,15 +21,16 @@ export const iqPriceApi = createApi({
   refetchOnMountOrArgChange: 60,
   refetchOnFocus: true,
   endpoints: (builder) => ({
-    getIqPrice: builder.query<
-      {
-        response?: any
-        status: boolean
-        message: string
-      },
-      string
-    >({
+    getIqPrice: builder.query<TResponseData, string>({
       query: (tokenName: string) => `?tokenName=${tokenName}`,
+      transformResponse: (response: TResponseData, _meta, arg: string) => {
+        const formatedResponse = response.response?.data?.[arg]
+        return {
+          response: formatedResponse,
+          status: response.status,
```

Comment:
Are we using this data somewhere?

Reply:
it's a refactor of how we are consuming the cmc local endpoint, it has a lot of repeated codes

Reply:
Yeah, I mean the status and message?